source/PowerUp.js

```diff
@@ -97,8 +97,286 @@ export class FiveShoota extends PowerUp {
 export class RapidFire extends PowerUp {
       constructor() {
           super()
-          this.fireRate = 80
-          this.bulletSpeed = 150
+          this.vector = new Victor(1,0)
+          this.bulletSpeed = 2
+          this.projectileType = "bullet"
+          this.rapidFire = true
+      }
+
+      fire(curShip) {
+
+          game.addChild(new Projectile(
+              curShip.position.x + curShip.width,
+              curShip.position.y + curShip.height/2,
+              this.vector, curShip, this.bulletSpeed, this.projectileType))
+      }
+
+}
+
+export class RapidSprayShot extends PowerUp {
+
+      constructor() {
+          super()
+          this.vector = new Victor(1,0)
+          this.vectorN = new Victor(1,0.25)
+          this.vectorP = new Victor(1,-0.25)
```

Comment:
What's the naming convention here? This is obviously the middle, top, and bottom projectiles, but what are the `N` and `P` denoting?